Marlin/src/lcd/language/language_hu.h

```diff
@@ -108,12 +111,15 @@ namespace Language_hu {
   PROGMEM Language_Str MSG_LASER_POWER                     = _UxGT("Lézer telj.");
   PROGMEM Language_Str MSG_SPINDLE_POWER                   = _UxGT("Orsó telj.");
   PROGMEM Language_Str MSG_LASER_TOGGLE                    = _UxGT("Lézer váltás");
+  PROGMEM Language_Str MSG_LASER_EVAC_TOGGLE               = _UxGT("Hütés váltás");
+  PROGMEM Language_Str MSG_LASER_ASSIST_TOGGLE             = _UxGT("Levegö segéd");
   PROGMEM Language_Str MSG_LASER_PULSE_MS                  = _UxGT("Impulzus teszt ms");
   PROGMEM Language_Str MSG_LASER_FIRE_PULSE                = _UxGT("Tüz impulzus");
+  PROGMEM Language_Str MSG_FLOWMETER_FAULT                 = _UxGT("Hütöfolyadék áramlási hiba");
```

Comment:
This is 7 characters too long to be visible on most screens. Can this be reduced to <= 20 characters?

Reply:
Thank you for the warning. I corrected.